game/scripts/npc/items/item_spirit_vessel.txt

```diff
@@ -76,7 +76,7 @@
       }
       "kill_charges"                                      "1"
       "soul_heal_amount"                                  "45 60 85 120 165" // Urn (30); vanilla (40);
-      "soul_damage_amount"                                "35 45 65 95 135" // Urn (25); vanilla (35);
+      "soul_damage_amount"                                "30 40 60 90 130" // OAA Urn (25); vanilla (35);
```

Comment:
Spirit Vessel is entirely custom so you dont need OAA here. Also is vanilla 35 dmg too much for lvl 1 spirit vessel in OAA?